cas/remote.go

```diff
@@ -60,9 +68,61 @@ func (r Remote) Type() int64 {
 	return remoteType
 }
 
+// Download downloads and verifies the remote ACI.
+// Download returns the signer, an *os.File representing the ACI, and an error if any.
+// err will be nil if the ACI downloads successfully and the ACI is verified.
+func (r Remote) Download(ks *keystore.Keystore, insecureSkipVerify bool) (*openpgp.Entity, *os.File, error) {
```

Comment:
What do you think about removing the insecureSkipVerify and inferring that based on whether or not the given `*keystore.Keystore` is nil? Simpler interface maybe? (also prevents those pesky nil pointer derefs ;-))